app/docs/docs/testing/e2e-testing.md

```diff
@@ -0,0 +1,93 @@
+# E2E Testing
+
+## Pattern
+
+For our e2e tests we are using a custom version of the [POM](https://playwright.dev/docs/pom) (Page Object Model). The idea is for the class to handle all the locators and expects, abstracting each step of the testing to make the tests themselves clear and easy to follow. We also want to design the tests in a user flow centric way, putting focus on ensuring a user can perform a given action without encountering issues along the way.
+
+## Landing Page Login Example
+
+We want to test that a user can log in from the landing page and be redirected to the correct protected route. Traditionally in playwright this would be done via direct playwright page interactions
+
+### Traditional Way
+
+```ts
+test('I can login from the app bar', async ({ page }) => {
+    await page.goto('/'),
+        await page.locator('text=Login').click(),
+        await page.locator('input[type="email"]').fill('test@prytaneum.io'),
+        await page.locator('input[type="password"]').click(),
+        await page.locator('input[type="password"]').fill('password'),
+        await Promise.all([
+            page.waitForNavigation(),
+            page.locator('div[role="dialog"] button:has-text("Login")').click(),
+        ]);
+    await expect(page).toHaveURL('/organizations/me');
+});
+```
+
+While this test does pass and work, it is not obvious at a glance what is going on with each step, and for longer, more complicated selectors, it can become even more difficult to follow.
+
+### Our Pattern
+
+Rather than putting all these page locators and interactions within the test, we can move them to a class based on the page we are testing. This way we can clearly dictate each step of our test in plain english, thus minimizing confusion.
+
+#### Landing Page Class
+
+When constructing we can locate all the components we want to interact with or ensure is visible/hidden for the user.
+As for the actions, they can all be represented as methods.
+NOTE: Even if the component is not currently in the DOM you can still set it in the constructor since the locator is dynamic.
+NOTE: Keep the methods small and in the scope of one specific task as shown in the example below. No one method should be perfomring multiple actions.
```

Comment:
Make these admonitions https://docusaurus.io/docs/next/markdown-features/admonitions

also, performing is misspelled in the second note.